mantidproject/mantid
# Peak position diagnostic tool

Added a new tool to the instrument view that allows the user to select two peaks and examine there properties and the angles between them.

**To test:**
- Load something that you can look at peaks with. I used WISH00035988.raw.
- Open the instrument view and select the pick tab
- Use the peak selection tool to choose some peaks (at least two)
- Choose two peaks on the interface: click one peak then click another.
- Observe the information about them displayed in the box on the left is correct

Fixes #17753 .

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
